internal_ws/ykcompile/src/lib.rs

```diff
@@ -640,15 +645,17 @@ impl TraceCompiler {
         );
     }
 
-    /// Push the specified registers to the stack in order.
+    /// Push the specified registers to the stack, in the order they are specified in the array.
```

Comment:
"order they are specified" -> "order they appear"

Reply:
Fixed in db5a334.